policy/Object.yaml

```diff
@@ -0,0 +1,381 @@
+funcs:
+    objectId: # the index of deepCoadd_obj IS the objectId
+        functor: Index
+    parentObjectId:
+        functor: Column
+        args: parent
+        dataset: ref
+    coord_ra:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        functor: RAColumn
+        dataset: ref
+    coord_dec:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        # coord_dec because "dec" is reserved in most SQL DBs
+        functor: DecColumn
+        dataset: ref
+    Ra:
+        functor: RAColumn
+        dataset: meas
+    Dec:
+        functor: DecColumn
+        dataset: meas
+    # raErr:  not available yet DM-15180
+    # decErr: not available yet DM-15180
+    refBand: # reference band is same for all measurements
+        functor: ReferenceBand
+        dataset: ref
+    # psRadecTai: No moving point source model yet.
+    # psRadec: No moving point source model yet.
+    # psPm: No moving point source model yet.
+    # psParallax: No moving point source model yet.
+    PsFlux:
+        functor: NanoJansky
+        filt: HSC-R
+        # Should we include forced_src too?
```

Comment:
If we have to pick one, I think including *just* the `forced_src` flux would be more appropriate; the model described by the DPDD uses the same position (and motion) across all bands.